index.html

```diff
@@ -0,0 +1,153 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+<!-- HTML File for Davins Portfolio-->
+  <title>Davin's Portfolio</title>
+  <link href="normalize.css" type="text/css" rel="stylesheet">
+  <link href="style.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+	<div class="mainContainerGrid">
+<!-- Header & Navigation -->
+		<header class="colorHeaderFooter fontCursive">
+			<nav class="fontSizeNav ">
+				<a href="#About">About</a>
+				<a href="#Portfolio">Portfolio</a>
+				<a href="#Skills">Skills</a>
+				<a href="#Contact">Contact</a>
+			</nav>
+		</header>
+
+		<main>
+<!-- Section Name & Background Picture -->			
+			<section class="colorBackgroundDark bkgImageCoding">
+				<!-- <h1 class="fontCursive colorFontSections">About</h1> -->
+				<!-- bkg img not working as expecte -->
+				<div class="containerFlex">
+					<article class="articleBox contentIntro">
+						<div>
+							<p class="textAboutMe fontCursive fontSizeMyName">Davin Hayden</p>
+						</div>
+						<div>
+							<p class="textAboutMe">Web Developer in UK & NZ</p>
+						</div>	
+					</article>
+					
+				</div>
+			</section>
+			
+<!-- Section About Me -->
+			<section class="colorBackgroundDark">
+				<a name="About">
+				<h1 class="fontCursive colorFontSections">About</h1>
```

Comment:
Shouldn't these be h2 with your name etc as the h1?

Reply:
I'm using h1 for the top level headings which I can reuse multiples times eg About, Portfolio etc. My name is being styled as an independent element not using h tag, as my name & its styling will not be reused multiple times on the same page